packages/playground/src/pages/sandbox-repo-import-page.tsx

```diff
@@ -0,0 +1,252 @@
+import React, { useEffect } from 'react'
+import {
+  Button,
+  ButtonGroup,
+  Input,
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+  Text,
+  Checkbox,
+  Spacer
+} from '@harnessio/canary'
+import { useForm, SubmitHandler } from 'react-hook-form'
+import { zodResolver } from '@hookform/resolvers/zod'
+import { z } from 'zod'
+import { FormFieldSet } from '../index'
```

Comment:
This is tricky because `FormFieldSet` or `SandboxLayout` are not exported in the component file. 

Currently SanboxLayout exports:
```
export { Root, LeftPanel, LeftSubPanel, Header, SubHeader, Main, Content, Columns, Column }
```
Similarly, form field set:

```
export { Root, Legend, SubLegend, Item, Label, ControlGroup, Caption, Message, Option, Separator, Spacer, MessageTheme }

```
And all of this is exported as `SandboxLayout` or `FormFieldSet` in `Index.ts`